From b8a311d5ea0332d0bf23693831744ff78f62f8ba Mon Sep 17 00:00:00 2001
From: Fancy Fang <chen.fang@nxp.com>
Date: Mon, 3 Jul 2017 14:58:46 +0800
Subject: [PATCH 2070/5242] MLK-15322-9 ARM64: dts: imx8mq-evk: add adv7535
 node under i2c1

commit  7298db3cea799f76248eade3f5c4fb8ae9606b3a from
https://source.codeaurora.org/external/imx/linux-imx.git

Add adv7535 device node under i2c1 on imx8mq evk board.

Signed-off-by: Fancy Fang <chen.fang@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8mq-evk.dts |   10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk.dts b/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk.dts
index 4525146..616c95d 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk.dts
@@ -269,6 +269,16 @@
 			};
 		};
 	};
+
+	adv7535: adv7535@3d {
+		compatible = "adi,adv7535";
+		reg = <0x3d>;		/* PD pin is low */
+		/* TODO: pin config & irq */
+		video-mode = <34>;	/* 1920x1080@30HZ */
+		dsi-traffic-mode = <0>;
+		bpp = <24>;
+		status = "okay";
+	};
 };
 
 &i2c2 {
-- 
1.7.9.5

